assets/js/components/notifications/UserInputSettings.js

```diff
@@ -84,12 +83,9 @@ export default function UserInputSettings( {
 		<BannerNotification
 			id={ `user-input-settings-notification-${ instanceID }` }
 			className="googlesitekit-user-input__notification"
-			title={ __(
-				'Customize Site Kit to match your goals',
-				'google-site-kit'
-			) }
+			title={ __( 'Key Metrics', 'google-site-kit' ) }
```

Comment:
I think we should stick with the casing in the Figma file, even if it's a bit inconsistent with the other titles on the page, as it would still be consistent with other notification banners in the plugin.

```suggestion
			title={ __( 'Key metrics', 'google-site-kit' ) }
```